guides/v2.1/config-guide/multi-site/ms_over.md

```diff
@@ -34,13 +36,15 @@ This section discusses the tasks required to use the `MAGE_RUN_TYPE` and `MAGE_R
 
 We use the following terms:
 
-*	*Website* is the top-level container for sites, shipping methods, payment methods, and so on. To create completely separate sites that do not share cart, shipping methods, and so on,  you must create separate websites. 
+*	*Website* is the top-level container for sites, shipping methods, payment methods, and so on. To create completely separate sites that do not share cart, shipping methods, and so on,  you must create separate websites.
+
+	Using  Website Customer account can be shared with different Website from a single magento instance. One Website contains at least one store. Catalog Prices should be manage at Website level.
```

Comment:
@RakeshJesadiya Why are you using capitalize in a random place? any reason?

Reply:
This paragraph is a little confusing. I would propose a rewrite similar to:

"Website customer accounts can be shared between multiple websites within a single Magento instance. A website contains at least one store. Catalog prices should be managed at the website level."

Reply:
Good content addition though!

Reply:
Done. Thanks for appreciating.